README.md

```diff
@@ -32,10 +32,14 @@ containing:
 ```python
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
+# rules_nodejs provides build rules related to NPM and Yarn package management.
```

Comment:
actually, rules_nodejs is more than this - any rules which are not typescript-specific go there. for example, there is `jasmine_node_test` and soon we'll merge `rollup_bundle`.

Reply:
Thanks - updated. I decided to still lead with the package management part since that seems like the primary use case.